modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/uiselect/plugin.js

```diff
@@ -90,6 +90,10 @@
 		base: CKEDITOR.event,
 
 		proto: {
+			setValue(value) {
+				this._setValue(value);
```

Comment:
I tried to use this way but it don't work.

`setValue` is defined internally somewhere but when using an exception is raised:

`Uncaught TypeError: Cannot set properties of undefined (setting 'value')`

When seeing instance.execute method inside render it works well but I don't know how to reach this part yet.

Reply:
@diegonvs been a while since I looked at these ckeditor plugins, but I think it's because you're returning `instance` [here](https://github.com/liferay-frontend/liferay-portal/blob/4d7f36d00096b0365152f832155d04e644bedad6/modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/uiselect/plugin.js#L158), which doesn't have set `setValue`, also `this` might not be the "this" you're looking for.

Reply:
> been a while since I looked at these ckeditor plugins, but I think it's because you're returning instance here, which doesn't have set setValue, also this might not be the "this" you're looking for.

Yes, We're returning the instance when calling render somewhere but I think it will not be helpful to me try to reach this code at this LOC(https://github.com/liferay-frontend/liferay-portal/pull/1472/files#r709661088). It runs before the component render, so, It's sad to me.

Today, I tried to make it work with a selectElement.selectedIndex trick as I have done before in TextHeight but it worked but again, as the line height thing, the state of the select it's not preserved and inside AlloyEditor it is preserved.

Reply:
Solved!